classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+  end
+
+  def add_student(student)
+    @students.push(student)
+    student.classroom = self
+  end
```

Comment:
- [ ] Kindly create a belongs-to relationship between `Classroom` and `Student` (a student belongs to a classroom) in the Student Class. You can refer to this [article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships) for examples.

Reply:
- [ ] The issue here has not been properly resolved. The changes are to be made in the Student class, kindly create the belongs-to relationship in the Student class. Example shown below, in this case `owner` should be `classroom` and `animals` should be `students`.

```ruby
  def owner=(owner)
    @owner = owner
    owner.animals.push(self) unless owner.animals.include?(self)
  end
```